contracts/swap/swap.go

```diff
@@ -46,13 +45,15 @@ type Backend interface {
 // Contract interface defines the methods exported from the underlying go-bindings for the smart contract
 type Contract interface {
 	// Withdraw attempts to withdraw Wei from the chequebook
-	Withdraw(auth *bind.TransactOpts, backend Backend, amount *big.Int) (*types.Receipt, error)
```

Comment:
Backend removed because it is not needed

Reply:
This still references `wei` in the comments

Reply:
Done